modules/ROOT/pages/mule-object-stores.adoc

```diff
@@ -0,0 +1,89 @@
+= Store Application Data Using Object Stores
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, object, store
+
+An object store is a facility for storing objects in or across Mule applications. Mule runtime engine(Mule) uses object stores to persist data for eventual retrieval. Internally, Mule uses object stores in various filters, routers, and other message processors that need to store states between messages.
+
+If you deploy your application to CloudHub, you can also use Object Store V2.
+
+== Use Cases
+
+You can configure a custom object store to use in the following scenarios:
+
+* When configuring an Idempotent Message Validator, to store the unique IDs in the specified object store.
+* When configuring a Redelivery policy in a listener(HTTP, File, JMS, VM, Socket), to store the redelivery count in the specified object store.
+* When configuring a Cache Scope that uses a custom caching strategy, to store the cached data in the specified object store.
+* When a custom component uses an object store to persist information.
+* When storing or retrieving information from a Mule flow, by using the Object Store Connector.
+
+== Configuration
+
+By default, each Mule application has an object store that is persistent and is always available to the app. You can also create custom object stores, which can be persistent or in-memory.
+
+You can configure a custom object store for a specific component that supports its usage, or configure the object store as a global element, to be accessible for any component in your app.
```

Comment:
The order here (specific component, global) does not match the order of the subsequent subtopics.
Either change one or the other so that the intro and topics list in the same order.